resources/src/jsMain/kotlin/dev/icerock/moko/resources/desc/PluralFormattedStringDesc.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.resources.desc
+
+import dev.icerock.moko.resources.PluralsResource
+import dev.icerock.moko.resources.provider.JsStringProvider
+
+actual class PluralFormattedStringDesc actual constructor(
+    val pluralsRes: PluralsResource,
+    val number: Int,
+    val args: List<Any>
+) : StringDesc {
+
+    override suspend fun localized(): String =
+        localized(pluralsRes.loader.getOrLoad())
+
+    override fun localized(provider: JsStringProvider): String {
+        return pluralsRes.localized(
+            provider = provider,
+            locale = StringDesc.localeType.locale,
+            quantity = number,
+            args = args.toTypedArray()
```

Comment:
here not missed star?

Reply:
no, since I references to this param by name. btw, everything compiles and works fine.